### PR TITLE
ARXIVNG-207 ARXIVNG-208 Reorganization

### PR DESCRIPTION
**Pull request by erickpeirson**
Ok, take a look at this as a first pass. Fire away.

### CODE REVIEW

**Comment by erickpeirson** [comment]
See also https://github.com/cul-it/arxiv-zero/pull/13

File: README.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+arXiv-NG services.
+
+Components of this project **must** meet all of the following criteria:
+
+1. It is likely that the component will be utilized in many or all arXiv
+  services.
+2. Once stable, it is unlikely to change often.
+3. It is unlikely that implementing new features in specific services
+   would require changes to the component.
+4. When a components does change, it **must** change in the same way for all of
```
Comment (eawoods) [inline README.md]: Nit picking: "When a component does change, it" or "When components do change, they"

**Comment by erickpeirson** [comment]
@eawoods I have a branch open in search that does just that ;-) Will raise a PR

**Comment by erickpeirson** [comment]
@bbarker good point; I just copied it over from zero thinking we'd want it later. I'll add the mypy.ini

**Comment by erickpeirson** [comment]
@JaimieMurdock @mhl10 Any comments before I merge?

**Comment by erickpeirson** [comment]
@eawoods Here's the PR for search: https://github.com/cul-it/arxiv-search/pull/22

**Pull request merged**